docs/guides/databases_with_concord.md

```diff
@@ -2,14 +2,134 @@
+        log_trace("[libpq] Almost connecting; negotiating SSL encryption.");
+        break;
+    case CONNECTION_SETENV:
+        /* Negotiating environment-driven parameter settings, soonly will be connected */
+        log_trace("[libpq] Almost connecting; negotiating environment-driven parameter settings.");
+        break;
+    default:
+        /* Ugh, something went wrong while connecting to the database, probably a wrong password, firewall, or something blocking the connection. */
+        log_fatal("[libpq] Error when trying to connect to the postgres database server. [%s]\n", PQerrorMessage(conn));
+        PQfinish(conn);
+}
+```
+
+### Creating a table
+
+We got the database connected, so now we can now proceed creating a table, since it's required before saving any data into it.
+
+```c
+
+/* Here we are using PQexec to execute commands, on this example, we are using "CREATE TABLE" command, which creates a table in the database, but with the `IF NOT EXISTS` clause, it will only create the table if it doesn't exist, and the table will have two collumns with the value INT. */
+PGresult *res = PQexec(conn, "CREATE TABLE IF NOT EXISTS concord_guides(user_id INT, guild_id INT);");
+
+if (PQresultStatus(res) != PGRES_COMMAND_OK) {
+    /* Something went wrong while creating the table, so we are going to log the error message of what went wrong. */
+    log_fatal("[libpq] Error when trying to create the table. [%s]", PQerrorMessage(conn));
+    PQclear(res);
+    /* Even not properly being executed, this isn't, at least most of the time, the fault of the connection, so we are not closing it. */
+    // PQfinish(conn);
+    return;
+} else {
+    log_debug("[libpq] Successfully created the table.");
+}
+```
+
+### Saving data into the table
+
+Now that we have a table, we can now proceed saving data into it, which is really easy to do, since it's almost equal to create a table, the only difference is the command we send to the database to be executed.
+
+```c
+/* Creating a row in the created table with the user_id and guild_id of the message author as collumns. */
+PGresult *res = PQexec(conn, "INSERT INTO concord_guides(user_id, guild_id) values(%"PRIu64", %"PRIu64");", msg->author->id, msg->guild_id);
+
+if (PQresultStatus(res) != PGRES_COMMAND_OK) {
+    /* Failed to insert the row, maybe the connection was blocked, or something like it. */
+    log_fatal("[libpq] Error when inserting a row. [%s]", PQerrorMessage(conn));
+    /* Deallocating the res struct made while trying to create the row. */
+    PQclear(res);
+    return;
+} else {
+    /* Successfully inserted the row (with the user_id and guild_id structures) into the table. */
+    log_debug("[libpq] Successfully inserted a row.");
+}
+```
+
+And done, we have a row in the table, but now we have to read it, proceed to the next section.
+
+### Reading data from the table
+
+Read the data is deadly easy, but it's a little bigger than the other functions, so let's see how to do it.
```

Comment:
Reading the data is dead easy, but it's a little bigger than the other functions.